projects/core/src/lib/talks/models/talk.ts

```diff
@@ -0,0 +1,9 @@
+export interface Talk {
+	lessonId: string;
+	result: string;
+	inProgress: boolean;
+	expertId: string;
+	menteeId: string;
```

Comment:
To my mind it will be very complicated to store in several places the same entities in database.
As we use serverless architecture, we need to manage it on frontend and build entities as less as possible.

So, it's a good point for discussion, maybe we can split entities like DTO and frontend, but for now let's keep it simple.